.github/workflows/actions.yml

```diff
@@ -0,0 +1,63 @@
+# 59 23 * * *
+
+name: Update natural gas data
+
+on:
+  # Schedule to run on the 1st day of each month
+  schedule:
+    - cron: '59 23 * * *'
+
+  # Run on push to master branch
```

Comment:
rename in the comments pls

Reply:
on the other hand, i think these comments are unnecessary because the yaml config is human-readable and self-describing.